app/models/computacenter/responsible_body_urns.rb

```diff
@@ -36,6 +36,8 @@ def computacenter_identifier
       when 'LocalAuthority'
         "LEA#{gias_id}"
       when 'Trust'
+        return '' if companies_house_number.blank?
```

Comment:
@tonyheadford Out of curiosity, why do we do this? Perhaps I've missed something in the doc.

Reply:
In these cases we're making up a responsible body to hang the school on - a school has to have an RB in the service.  We may not be able to find a suitable companies house number for the school (as it is a Trust attribute) so we have to handle the case where it could be blank.  The companies house number is used as an identifier for CC (I don't know the exact reason why), they have agreed that we could send them blank ones for these.  Does that help, not sure I'm answering the right question!